src/config-schema.ts

```diff
@@ -453,7 +453,7 @@ Front matter is hidden by default, but you can choose between \`table\` (like [o
   usePuppeteerCore: {
     title: "Use puppeteer-core",
     description:
-      "If set to true, then locally installed puppeteer-core will be required. Otherwise, the puppeteer globally installed by `npm install -g puppeteer` will be required.",
+      "If set to true, then locally installed puppeteer-core will be required. Otherwise, the puppeteer globally installed by `yarn install -g puppeteer` will be required.",
```

Comment:
@kachkaev this one is probably not correct? Might still be `npm install -g puppeteer` or `yarn global add puppeteer`?

Reply:
Ooops you are correct about this one: should be `npm install -g` (because it's global).

Reply:
Fixed